config/configgrpc/configgrpc.go

```diff
@@ -127,7 +127,7 @@ type GRPCServerSettings = ServerConfig
 // ServerConfig defines common settings for a gRPC server configuration.
 type ServerConfig struct {
 	// Server net.Addr config. For transport only "tcp" and "unix" are valid options.
-	NetAddr confignet.NetAddr `mapstructure:",squash"`
+	AddrConfig confignet.AddrConfig `mapstructure:",squash"`
```

Comment:
The name of the variable may make sense to be `NetAddrConfig` (not the type since the type is referenced with the package).

Reply:
I removed all field renaming from this PR